.golangci.yaml

```diff
@@ -62,6 +62,7 @@ linters:
     - govet
     - ineffassign
     - staticcheck
+    - varcheck
```

Comment:
nit: varcheck now can be removed from below list _To enable later if makes sense_

Reply:
Let's sort it in this PR, maybe?

Reply:
Done.